test/jdk/java/lang/management/OperatingSystemMXBean/GetSystemLoadAverage.java

```diff
@@ -52,31 +58,48 @@ public class GetSystemLoadAverage {
 
     // The system load average may be changing due to other jobs running.
     // Allow some delta.
-    private static double DELTA = 0.05;
-
-    public static void main(String args[]) throws Exception {
-        if (args.length > 1)  {
-            throw new IllegalArgumentException("Unexpected number of args " + args.length);
-        }
-
-        if (args.length == 0) {
-            // On Linux or Solaris
-            checkLoadAvg();
-        } else {
-            // On Windows, the system load average is expected to be -1.0
-            if (!args[0].equals("-1.0")) {
-                throw new IllegalArgumentException("Invalid argument: " + args[0]);
-            } else {
-                double loadavg = mbean.getSystemLoadAverage();
-                if (loadavg != -1.0) {
-                    throw new RuntimeException("Expected load average : -1.0" +
-                        " but getSystemLoadAverage returned: " +
-                        loadavg);
+    private static final double DELTA = 0.05;
+
+    private static final int MAX_RETRIES = 5;
+    private static final int WAIT_TIME_MS = 5000;
+
+    @Test(timeOut = (300 + WAIT_TIME_MS) * MAX_RETRIES)
+    void testSystemLoadAvg() throws Exception {
+        for (int i = 1; i <= MAX_RETRIES; i++) {
+            try {
+                System.out.println(
+                        String.format("Run %d: TestSystemLoadAvg", i));
+                if (!Platform.isWindows()) {
+                    // On Linux or Solaris
```

Comment:
Sorry, if missed the first time. It is Linux or Mac, or better something like *mix.